Выполнил задание по спринту №4

bakytushka opened the pull request:
Получилось выполнить задание по спринту №4 в полном объеме. Особых трудностей не возникло.

bakytushka committed:
Выполнил задание по спринту №4

So1ovyov commented on MovieQuiz/Helpers/UIColor+Extensions.swift:
```diff
@@ -1,3 +1,10 @@
 import UIKit
 
-extension UIColor { }
+extension UIColor {
```
Кажется сместились отступы:
```
extension UIColor {
    static var ypBlack: UIColor { UIColor(named: "YP Black") ?? UIColor.black }
    static var ypGreen: UIColor { UIColor(named: "YP Green") ?? UIColor.green }
    static var ypRed: UIColor { UIColor(named: "YP Red") ?? UIColor.red }
    static var ypBackground: UIColor { UIColor(named: "YP Background") ?? UIColor.darkGray }
    static var ypGray: UIColor { UIColor(named: "YP Gray") ?? UIColor.gray }
    static var ypWhite: UIColor { UIColor(named: "YP White") ?? UIColor.white}
}
```

So1ovyov commented on MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -1,10 +1,180 @@
 import UIKit
 
+struct QuizQuestion {
```
У структур также можно добавить модификатор `private`

So1ovyov commented on MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -1,10 +1,180 @@
+    @IBOutlet private weak var textLabel: UILabel!
+    @IBOutlet private weak var counterLabel: UILabel!
+    
+    private var currentQuestionIndex = 0
```
Если есть желание ты можешь использовать синтаксический сахар языка swift 
`private var currentQuestionIndex: Int = .zero`

So1ovyov commented on MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -1,10 +1,180 @@
+    
+    
+    private func showAnswerResult(isCorrect: Bool) {
+        yesButton.isEnabled = false
```
Если вдруг ты захочешь, чтобы строчки не повторялись
```
yesButton.isEnabled
noButton.isEnabled
```
тогда можно создать метод, который будет менять сразу у двух кнопок состояние включена/отключена
```
    private func changeButtonsState(isEnabled: Bool) {
        yesButton.isEnabled = isEnabled
        noButton.isEnabled = isEnabled
    }
```

bakytushka committed:
1. добавил скругление постера и рамки вокруг него
2. добавил метод изменения состояния сразу двух кнопок включена/отключена
3. добавил модификатор private структурам
4. поправил отступы
